server/service/client_apple_mdm.go

```diff
@@ -80,17 +80,17 @@ func (c *Client) MDMGetCommandResults(commandUUID string) ([]*fleet.MDMCommandRe
 	return responseBody.Results, nil
 }
 
-func (c *Client) MDMAppleListCommands() ([]*fleet.MDMAppleCommand, error) {
+func (c *Client) MDMListCommands() ([]*fleet.MDMCommand, error) {
```

Comment:
If you feel like it, this would now belong in a platform-agnostic `client_mdm.go` file.